config/deployer.sample.toml

```diff
@@ -138,3 +138,5 @@ InstanceEngine = 'aurora-postgresql'
 InstanceType = 'db.r6g.large'
 Password = 'mostest80098bigpass_'
 UserName = 'mmuser'
+
+[CustomTags]
```

Comment:
Unrelated but I have to ask; is there anyone using this format? Why was it introduced?

Reply:
Felipe added it. It's just for convenience I think more than anything.

Reply:
Yup, this was a proposal from @fmartingr, which we gladly accepted as a potential UX improvement. I am not using it at the moment, but I want to test it and feel the difference :) Not sure if Felipe is using it currently.